File: src/main/java/ru/mydesignstudio/database/metadata/extractor/extractors/model/DatabaseMetadata.java
package ru.mydesignstudio.database.metadata.extractor.extractors.model;

import java.util.Collection;
import lombok.Data;
import ru.mydesignstudio.database.metadata.extractor.extractors.job.JobModel;
import ru.mydesignstudio.database.metadata.extractor.extractors.procedure.ProcedureModel;
import ru.mydesignstudio.database.metadata.extractor.extractors.procedure.argument.ArgumentModel;

@Data
public class DatabaseMetadata {
  private String schemaName;
  private Collection<ProcedureModel> procedures;
  private Collection<ArgumentModel> arguments;
  private Collection<JobModel> jobs;
}

File: extractor-core/src/main/kotlin/ru/mydesignstudio/database/metadata/extractor/extract/result/DatabaseMetadata.kt
package ru.mydesignstudio.database.metadata.extractor.extract.result

class DatabaseMetadata(
        val schemaName: String,
        val procedures: Collection<ProcedureModel>,
        val arguments: Collection<ArgumentModel>,
        val tables: Collection<TableMetadata>,
        val jobs: Collection<JobModel>)